### ISSUE TITLE
Getting an unable to delete git-lfs.exe error 

### ISSUE THREAD

**Issue by Rod-in-NM** [git-for-windows/git#4293]
- [X] I was not able to find an [open](https://github.com/git-for-windows/git/issues?q=is%3Aopen) or [closed](https://github.com/git-for-windows/git/issues?q=is%3Aclosed) issue matching what I'm seeing

### Setup

 - Which version of Git for Windows are you using? Is it 32-bit or 64-bit?
64-bit
```
$ git --version --build-options

git version 2.39.2.windows.1
cpu: x86_64
built from commit: a82fa99b36ddfd643e61ed45e52abe314687df67
sizeof-long: 4
sizeof-size_t: 8
shell-path: /bin/sh
feature: fsmonitor--daemon

```

 - Which version of Windows are you running? Vista, 7, 8, 10? Is it 32-bit or 64-bit?

```
$ cmd.exe /c ver

Microsoft Windows [Version 10.0.18363.2274]
```

 - What options did you set as part of the installation? Or did you choose the
   defaults?

```
# One of the following:
> type "C:\Program Files\Git\etc\install-options.txt"
> type "C:\Program Files (x86)\Git\etc\install-options.txt"
> type "%USERPROFILE%\AppData\Local\Programs\Git\etc\install-options.txt"
> type "$env:USERPROFILE\AppData\Local\Programs\Git\etc\install-options.txt"
$ cat /etc/install-options.txt

Editor Option: VisualStudioCode
Custom Editor Path: 
Default Branch Option: main
Path Option: Cmd
SSH Option: OpenSSH
Tortoise Option: false
CURL Option: OpenSSL
CRLF Option: CRLFAlways
Bash Terminal Option: MinTTY
Git Pull Behavior Option: Merge
Use Credential Manager: Enabled
Performance Tweaks FSCache: Enabled
Enable Symlinks: Disabled
Enable Pseudo Console Support: Disabled
Enable FSMonitor: Disabled

```

 - Any other interesting things about your environment that might be related
   to the issue you're seeing?

Here is the error that I got while trying to run the 2.39.2 Git installer:

### Details
![Screenshot 2023-02-16 134420](https://user-images.githubusercontent.com/13875530/219485572-81b15e7a-ba95-4e49-804c-7a48b720d24f.jpg)

 - Which terminal/shell are you running Git from? e.g Bash/CMD/PowerShell/other

PowerShell

 - What commands did you run to trigger this issue? If you can provide a
   [Minimal, Complete, and Verifiable example](http://stackoverflow.com/help/mcve)
   this will help us understand the issue.

```
** insert your commands here **
```
 - What did you expect to occur after running these commands?

I expected the Git 2.39.2 to remove the older version of Git, then install Git 2.39.2. I wasn't aware of the fact that running the installer also removed git-lfs.exe.

 - What actually happened instead?

Please see the embedded picture

 - If the problem was occurring with a specific repository, can you provide the
   URL to that repository to help us with testing?

No

** insert URL here **

**Comment by dscho** [comment]
Such an error message usually means that a `git-lfs.exe` process was still running. Have you checked that?

**Comment by Rod-in-NM** [comment]
No, but on the other hand I don't know what would trigger `git-lfs.exe` to run. In fact, I hadn't been using it much, when I encountered the error. I'm in the process of setting up a new laptop (new to me) starting yesterday, and I'd only installed Git on the laptop yesterday.
